docs/guides/nrfconnect_platform_overview.md

```diff
@@ -0,0 +1,37 @@
+# nRF Connect Platform Overview
+
+The nRF Connect platform is a [CHIP](https://github.com/project-chip/connectedhomeip) platform using Nordic Semiconductor's [nRF Connect SDK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html) and [Zephyr RTOS](https://zephyrproject.org/).
+
+The nRF Connect SDK allows to build a range of cellular IoT (LTE-M and NB-IoT), Bluetooth Low Energy, Thread, Zigbee, and Bluetooth Mesh applications. It containes samples, libraries and full set of drivers for Nordic Semiconductor [nRF9160](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160), [nRF5340](https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF5340-PDK) and [nRF52 Series](https://www.nordicsemi.com/Products/Low-power-short-range-wireless) devices.
+
+The nRF Connect SDK is based around Zephyr RTOS, which is scalable real-time operating system, dedicated for the connected, resource constrained devices. Zephyr supports multiple hardware platforms, provides hardware drivers, application protocols, protocol stacks and even more. Besides of the Zephyr, nRF Connect SDK also intergrates other projects like crypto library mbed TLS, MCU Bootloader or Thread stack implementation [OpenThread](https://openthread.io/).
+
+The diagram presented below, shows simplified structure of the CHIP application running on the nRF Connect platform and using BLE and Thread stacks for communication purposes: 
+
+![nrfconnect overview](../../examples/platform/nrfconnect/doc/images/nrfconnect_overview.png)
```

Comment:
Image looks much better, thanks! 👍 

Small nits (sorry 😄 ):

- Resize IPv6 block to be under both UDP and TCP - full width
- There is a larger gap between Thread Stack and IP, and Thread start and Zephyr integration (visualization issue)
- You will kill me, but border around CHIP stack and Zephyr integration.. is very strong. Can we do sth more smooth and lightweight? 👍

Reply:
I working on figuring out how to fetch and merge your nits into the .png image using git :smile: If it comes to the border, do you mean only its thickness or color is also too strong?